Integrations/integration-SplunkPy

```diff
@@ -70,12 +80,55 @@ script:
         incident['labels'] = labels
         return incident
 
+    def handler(proxy):
+        proxy_handler = urllib2.ProxyHandler({'http': proxy, 'https': proxy})
+        opener = urllib2.build_opener(proxy_handler)
+        urllib2.install_opener(opener)
+        return request
+
+    def request(url, message, **kwargs):
+        method = message['method'].lower()
+        data = message.get('body', "") if method == 'post' else None
+        headers = dict(message.get('headers', []))
+        req = urllib2.Request(url, data, headers)
+        try:
+            response = urllib2.urlopen(req)
+        except urllib2.URLError, response:
+            # If running Python 2.7.9+, disable SSL certificate validation and try again
+            if sys.version_info >= (2, 7, 9):
+                response = urllib2.urlopen(req, context=ssl._create_unverified_context())
+            else:
+                raise
+        except urllib2.HTTPError, response:
+            pass # Propagate HTTP errors via the returned response message
+        return {
+            'status': response.code,
+            'reason': response.msg,
+            'headers': response.info().dict,
+            'body': StringIO(response.read())
+        }
+
     service = client.connect(
             host=demisto.params()['host'],
             port=demisto.params()['port'],
             username=demisto.params()['authentication']['identifier'],
             password=demisto.params()['authentication']['password'])
 
+    proxy = demisto.params()['proxy']
+    if proxy:
+        try:
+            service = client.connect(
```

Comment:
Used similar code to the original python SDK example:
https://github.com/splunk/splunk-sdk-python/blob/master/examples/handlers/handler_proxy.py

I prefer to keep it like this, so it can be referenced easily